src/modules/Game.class.js

```diff
@@ -20,49 +20,255 @@ class Game {
    * If passed, the board will be initialized with the provided
    * initial state.
    */
+
   constructor(initialState) {
-    // eslint-disable-next-line no-console
-    console.log(initialState);
+    this.board = initialState || [
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+    ];
+    this.addRandomTile = this.addRandomTile.bind(this);
+    this.start = this.start.bind(this);
+    this.status = 'idle';
+    this.score = 0;
   }
 
-  moveLeft() {}
-  moveRight() {}
-  moveUp() {}
-  moveDown() {}
+  start() {
+    this.status = 'playing';
+    this.addRandomTile();
+  }
 
-  /**
-   * @returns {number}
-   */
-  getScore() {}
+  restart() {
+    this.status = 'playing';
+    this.score = 0;
 
-  /**
-   * @returns {number[][]}
-   */
-  getState() {}
+    this.board = [
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+    ];
 
-  /**
-   * Returns the current game status.
-   *
-   * @returns {string} One of: 'idle', 'playing', 'win', 'lose'
-   *
-   * `idle` - the game has not started yet (the initial state);
-   * `playing` - the game is in progress;
-   * `win` - the game is won;
-   * `lose` - the game is lost
-   */
-  getStatus() {}
+    this.addRandomTile();
+  }
 
-  /**
-   * Starts the game.
-   */
-  start() {}
+  cellMerge() {
+    let moved = false;
 
-  /**
-   * Resets the game.
-   */
-  restart() {}
+    for (let row = 0; row < this.board.length; row++) {
+      // Видаляємо нулі (пусті комірки)
```

Comment:
it is not good practice to left comments.
Code should be "self commented". It means that style of code, naming variables, functions and so on allow to read code and understand what code to do without comments.
General practice to write comments in english.